### PR TITLE
Clear relevant services when removing bricks

### PR DESCRIPTION
**Pull request by turbochef**
## What does this PR do?

- Fixes 5396

## Demo

- [Loom](https://www.loom.com/share/6a15ca477e85473abcc54b21d7879207)

## Checklist

- [x] Add tests
- [x] Designate a primary reviewer

### CODE REVIEW

**Comment by turbochef** [comment]
Had some trouble adding relevant tests because I couldn't reproduce the correct state in the test environment.

File: src/pageEditor/slices/editorSlice.ts [hunk 1]
```diff
@@ -732,6 +736,7 @@ export const editorSlice = createSlice({
         pipelineIndex: number;
       }>
     ) {
+      console.log({ testaction: action });
```
Comment (twschiller) [inline src/pageEditor/slices/editorSlice.ts:739]: Remember to remove before merging

Comment (turbochef) [reply src/pageEditor/slices/editorSlice.ts]: Thanks for catching it. We could consider adding a linter rule to prevent console.logs.

**Review by BLoe — APPROVED** [review]
🚀  lgtm

**Comment by twschiller** [comment]
🚢 :shipit: 

image

**Pull request merged**